include/constants/battle_config.h

```diff
@@ -145,6 +145,7 @@
 #define B_PARALYZE_ELECTRIC         GEN_6 // In Gen6+, Electric-type Pokémon can't be paralyzed.
 #define B_POWDER_GRASS              GEN_6 // In Gen6+, Grass-type Pokémon are immune to powder and spore moves.
 #define B_STEEL_RESISTANCES         GEN_6 // In Gen6+, Steel-type Pokémon are no longer resistant to Dark and Ghost moves.
+#define B_TRAINERS_KNOCK_OFF_ITEMS  TRUE  // If TRUE, trainers can steal/swap your items (non-berries are restored after battle). In vanilla games trainers cannot steal items.
```

Comment:
Comment is incorrect as trainers can steal items in vanilla, just not permanently unless the item was consumed by them. I noted that the setting being TRUE matches the vanilla behaviour when testing for my previous review.